data/adapters/gersbach_E2G_CRISPR_adapter.py

```diff
@@ -108,6 +161,7 @@ def process_file(self):
                         'biological_context': biosample_term,
```

Comment:
is this an array right now? think we agreed to have this field as str if possible

Reply:
You're right, I can adjust it to a string. I think the Perturb-seq data may have to be reloaded then.